### ISSUE TITLE
[Minor][lunarsoil_big_world] Accessibility option to turn off flashes and other visuals

### ISSUE THREAD

**Issue by morgondag** [MorgondagDev/lunar-soil-tracker#109]
![https://lunar-soil-screenshots.morgondag.io/0a2692c94233a19820a81088ecdf7b2b.png](https://lunar-soil-screenshots.morgondag.io/0a2692c94233a19820a81088ecdf7b2b.png)

**💬Description:**
I was travelling pretty far in the opposite direction (the windy dust area) and noticed that there was a flash followed by an abrupt noise.  

I love the accessibility options so far, but I think that flashing lights (especially ones that are not announced) should be turned off for users with photosensitive issues.

I suggest that there be a toggle option in Settings that can turn off flashing, as well as glare.

**⚙️Ticket-ID:** 0a2692c94233a19820a81088ecdf7b2b

👨‍🚀: WINSEVN

**⚠️Severity:** Minor

**🗺️Level:** lunarsoil_big_world

**📍Player Position:**
X=175586.072 Y=226257.140 Z=28659.822

**⏲️Gametime & FPS Delta**
1949.906154 - 21.869731903076172

---

⚙️ 0.0.7.1 - 5.2.0-25360045+++UE5+Release-5.2 - 76561198853483566 - 
⚙️ GenericGPUBrand - 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz 
*⚙️Generated from a in-game report*
*⏲️Sun Jun 04 2023 04:13:33 GMT+0000 (Coordinated Universal Time)*
